docs/excel/excel-add-ins-charts-data-labels.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Work with data labels in charts using the Excel JavaScript API
+description: Code samples demonstrating chart data label tasks using the Excel JavaScript API.
+ms.date: 04/14/2025
+ms.localizationpriority: medium
+---
+
+# Work with data lables in charts using the Excel JavaScript API
+
+Add data labels to Excel charts to provide a better visualization experience about important aspects of the chart.
```

Comment:
I think including a support link will help provide context for the feature, for a reader that's like "what's a data label?" 
```suggestion
Add data labels to Excel charts to provide a better visualization experience about important aspects of the chart. To learn more about data labels, see [Add or remove data labels in a chart](https://support.microsoft.com/office/add-or-remove-data-labels-in-a-chart-884bf2f1-2e29-454e-8b42-f467c9f4eb2d).
```